extensions/wikia/LanguageWikisIndex/LanguageWikisIndexHooks.class.php

```diff
@@ -41,6 +41,18 @@ public static function onClosedWikiPage( $requestUrl ) {
 		return true;	// proceed with rendering the closed wiki page
 	}
 
+	public static function onGetHTMLBeforeWikiaPage( &$beforeWikiaPageHtml ) {
+		$path = parse_url( RequestContext::getMain()->getRequest()->getRequestURL(), PHP_URL_PATH );
+		if ( self::isEmptyDomainWithLanguageWikis() && $path === self::WIKIS_INDEX_PAGE ) {
+			$beforeWikiaPageHtml .= Html::element(
+				'header',
+				[ 'class' => 'wds-community-header' ],
+				''
```

Comment:
Empty element? What is it for? Does it look good?

Reply:
It's the community header without everything else in it. It puts the same block at the top that we will set the 404 background image on (once we have that asset) as shown in the designs.

Reply:
I get how this makes things easier but it can backfire when the community header styles are updated and we need to remember that it's used in a hacky way here. I'd prefer to copy the styles that we need just for this element.

Reply:
Good point, my concern was that the community header styles would be updated and we'd forget to do the same update here, as we'd want this to look the same as any new community header changes too, but it's better for it to look inconsistent if we make changes than potentially break. :+1:

Reply:
Done.